.github/workflows/enos-verify-stable.yml

```diff
@@ -4,10 +4,12 @@ on:
   repository_dispatch:
     types:
       - enos-verify-stable
+      - enos-verify-stable::*
 
 jobs:
   enos-verify-stable:
     name: Enos verify stable artifact
+    if: ${{ contains(github.event.client_payload.payload.branch, 'release') }}
```

Comment:
Nit: since ``contains`` will be true for the word ``release`` anywhere in the branch name, would we want to match the beginning of the branch name instead? (``startsWith ``)

Reply:
Totally makes sense 👍 !  Will update it now.  Thanks